src/lib/Toast/Toast.svelte

```diff
@@ -31,9 +78,11 @@
 
 {#if showToast}
   <div
-    class="toast {properties.type ?? ''}"
-    in:fly={{ x: 500, y: 0, duration: 400 }}
-    out:fly={{ x: 500, y: 0, duration: 800 }}
+    class="toast {properties.type ?? ''} {properties?.overContent === false
+      ? 'insideContent'
```

Comment:
lets add separate class check for neatness.

`class:overlap-content={...check_for_should_overlap...}`